src/App.css

```diff
@@ -1,13 +1,12 @@
 .result {
   background-color: grey;
-  width: 45%;
-  position: fixed;
+  width: 51.5%;
   height: 5rem;
   align-content: flex-end;
 }
 
 p {
-  margin-left: 30rem;
+  margin-left: 27rem;
   font-size: 30px;
   bottom: 0;
 }
```

Comment:
- [ ] The contents of the `p-tag` is overflowing out of the parent component. I think it is related to the `margin-left` you give. It looks too much.